plugin/kubernetes/kubernetes.go

```diff
@@ -292,7 +292,13 @@ func endpointFQDN(ep *object.Endpoints, zone string, endpointNameMode bool) []st
 	var names []string
 	for _, ss := range ep.Subsets {
 		for _, addr := range ss.Addresses {
-			names = append(names, dnsutil.Join(endpointHostname(addr, endpointNameMode), serviceFQDN(ep, zone)))
+			var namespace string
+			if addr.TargetRefNamespace != "" {
+				namespace = addr.TargetRefNamespace
+			} else {
+				namespace = ep.Namespace
+			}
+			names = append(names, dnsutil.Join(endpointHostname(addr, endpointNameMode), ep.GetName(), namespace, Svc, zone))
```

Comment:
This seems to remove the only used of serviceFQDN? If so we can get rid of that as well

Reply:
Hmm - actually - I'm guessing things got refactored at some point in the past, and _ALL_ of these FQDN functions were abandoned/orphaned ... Only used in tests.

So, obviously this PR is invalid.  I'll close it...